index.html

```diff
@@ -17,128 +18,277 @@
-               <li><a href="https://www.linkedin.com/in/abiy-hussen-aman-194587183/"><img src="Images\Vector2.svg" alt="" width="20px" height="20px"></a></li>
-               <li><a href="#"><img src="Images\Vector3.svg" alt="" width="20px" height="16px"></a></li>
-               <li><a href="https://github.com/abiy006"><img src="Images\Vector4.svg" alt="" width="17.99px" height="20px"></a></li>
+               <li><a href="#"><img src="Images\Vector4.svg" alt="" width="17.99px" height="20px"></a></li>
+               <li><a href="#"><img src="Images\Vector2.svg" alt="" width="20px" height="20px"></a></li>
                <li><a href="#"><img src="Images\Vector5.svg" alt="" width="14.17px" height="20px"></a></li>
+               <li><a href="#"><img src="Images\Vector1.svg" alt="" width="19.99px" height="19.96px"></a></li>
+               <li><a href="#"><img src="Images\Vector3.svg" alt="" width="20px" height="16px"></a></li>
             </ul>
          </div>
-    </section>
-    <section class="card-section">
-        <section class="card-1">
-            <div class="card-1-bg"></div>
-            <h2>Tonic</h2>
-            <div class="canopy"></div>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <div class="lang-btn">
+         <button>Get my resume</button>
+      <div class="About-languages">
+            <ul>
+               <li class="about-lang-menu"><a href="#"><button><h3>Languages</h3><img src="Images\about_down_arrow.svg" alt="" width="14px" height="8px"></button></a></li>
                <ul>
-                  <li><a href="#"><button>html</button></a></li>
-                  <li><a href="#"><button>css</button></a></li>
-                  <li><a href="#"><button>javaScript</button></a></li>
+                  <li><a href="#"><button><img src="Images\about_javascript2.svg" alt="" width="48px" height="48px"><h3>JavaScript</h3></button></a></li>
+                  <li><a href="#"><button><img src="Images\about_html2.svg" alt="" width="48px" height="48px"><h3>HTML</h3></button></a></li>
+                  <li><a href="#"><button><img src="Images\about_css2.svg" alt="" width="48px" height="48px"><h3>CSS</h3></button></a></li>
                </ul>
+               <li class="about-lang-menu"><a href="#"><button><h3>Frameworks</h3><img src="Images\about_side_arrow.svg" alt="" width="8px" height="14px"></button></a></li>
+               <li class="about-lang-menu"><a href="#"><button><h3>Skills</h3><img src="Images\about_side_arrow.svg" alt="" width="8px" height="14px"></button></a></li>
+            </ul>
+      </div>
+      </section>
+      <section class="contact_me_mobile">
+         <h2>Contact me</h2>
+         <p>If you have an application you are intersted in developing, a feature that you need built or a project that
+            needs coding. I'd love to help with it</p>
+         <form action="https://formspree.io/f/mbjezjkw" method="post">
```

Comment:
- [ ] Please make sure the contact form is centered (on screen size <768px) to keep the design consistent. This will make your project way more prettier which will improve the user experience 📱  😄 👍 

**Your project  design**
![image](https://user-images.githubusercontent.com/74411135/229288221-a0cee9c0-5b6d-43aa-9250-3542ec6dd533.png)

**Figma design**
![image](https://user-images.githubusercontent.com/74411135/229288234-51f62164-ca6c-46fc-8904-420add5954a1.png)